pkg/auth/requests/authenticate.go

```diff
@@ -302,3 +354,31 @@ func (a *tokenAuthenticator) TokenFromRequest(req *http.Request) (*v3.Token, err
 
 	return storedToken, nil
 }
+
+// Given a stored token with hashed key, check if the provided (unhashed) tokenKey matches and is valid
+func extVerifyToken(storedToken *ext.Token, tokenName, tokenKey string) (int, error) {
+	invalidAuthTokenErr := errors.New("Invalid auth token value")
```

Comment:
The error should not start with a capital letter.
```suggestion
	invalidAuthTokenErr := errors.New("invalid token")
```